Mathlib/CategoryTheory/Enriched/Limits/HasConicalLimits.lean

```diff
@@ -85,22 +84,43 @@ end Definitions
 
 section Results
 
-variable (J : Type u₁) [Category.{v₁} J]
+variable {J : Type u₁} [Category.{v₁} J] {J' : Type u₂} [Category.{v₂} J']
 variable (V : Type u') [Category.{v'} V] [MonoidalCategory V]
-variable (C : Type u) [Category.{v} C] [EnrichedOrdinaryCategory V C]
+variable {C : Type u} [Category.{v} C] [EnrichedOrdinaryCategory V C]
 
 /-- ensure existence of a conical limit implies existence of a limit -/
 example (F : J ⥤ C) [HasConicalLimit V F] : HasLimit F := inferInstance
 
+/-- If a functor `F` has a conical limit, so does any naturally isomorphic functor. -/
+lemma HasConicalLimit.of_iso {F G : J ⥤ C} [HasConicalLimit V F] (e : F ≅ G) :
+    HasConicalLimit V G where
+  toHasLimit := hasLimit_of_iso e
+  preservesLimit_eCoyoneda X := preservesLimit_of_iso_diagram (eCoyoneda V X) e
+
+instance HasConicalLimit.of_equiv (F : J ⥤ C) [HasConicalLimit V F]
+    (G : J' ⥤ J) [G.IsEquivalence] : HasConicalLimit V (G ⋙ F) where
+
+/-- If a `G ⋙ F` has a limit, and `G` is an equivalence, we can construct a limit of `F`. -/
+lemma HasConicalLimit.of_equiv_comp (F : J ⥤ C) (G : J' ⥤ J) [G.IsEquivalence]
+    [HasConicalLimit V (G ⋙ F)] : HasConicalLimit V F :=
+  have e : G.inv ⋙ G ⋙ F ≅ F := G.asEquivalence.invFunIdAssoc F
```

Comment:
I was wondering if `G.asEquivalence.invFunIdAssoc` is the canonical way or if there is a shorter term for `G.inv ⋙ G ⋙ F ≅ F`?